lib/ansiblelint/rules/AlwaysRunRule.py

```diff
@@ -18,7 +18,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-from ansiblelint import AnsibleLintRule
+from ansiblelint.rules import AnsibleLintRule
```

Comment:
I'd like to see relative imports in rules instead:
```suggestion
from . import AnsibleLintRule
```

Reply:
Why? Can you please show where is the use of relative imports indicated as a better coding practice?

Also, why to change import types as part of this change? This looks outside the scope of current change.

Reply:
Absolute imports may hit places outside of this project while relative imports point to the target modules explicitly. It's in the scope of this PR because you added a bunch of imports all around the place.

Reply:
It seems that you missed to spot that use of relative imports will break [load_plugins](https://github.com/ansible/ansible-lint/blob/7cffde295e238f13f563ebdeb1d4722589353fcb/lib/ansiblelint/utils.py#L129) and triger something like:
```
Traceback (most recent call last):
File "/Users/ssbarnea/.pyenv/versions/3.8.2/bin/ansible-lint", line 11, in <module>
load_entry_point('ansible-lint', 'console_scripts', 'ansible-lint')()
File "/Users/ssbarnea/c/os/ansible-lint/lib/ansiblelint/__main__.py", line 66, in main
rules = RulesCollection(rulesdirs)
File "/Users/ssbarnea/c/os/ansible-lint/lib/ansiblelint/rules/__init__.py", line 136, in __init__
self.extend(ansiblelint.utils.load_plugins(rulesdir))
File "/Users/ssbarnea/c/os/ansible-lint/lib/ansiblelint/utils.py", line 137, in load_plugins
spec.loader.exec_module(module)
File "<frozen importlib._bootstrap_external>", line 783, in exec_module
File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
File "/Users/ssbarnea/c/os/ansible-lint/lib/ansiblelint/rules/RoleNames.py", line 23, in <module>
from .roles import AnsibleLintRule
ImportError: attempted relative import with no known parent package
```

So I would defer switching from absolute imports to relative ones for a different change. At this moment this keeps the change at minimum: only points to the newer location of the AnsibleLintRule class.

In fact due to the introduction of the shim function, the code would run even without touching the import but it would raise a DeprecationWarning.

Reply:
Agreed. I didn't remember that we have our own import logic there.